src/main/java/io/vlingo/wire/fdx/bidirectional/netty/server/NettyInboundHandler.java

```diff
@@ -126,21 +124,19 @@ public void respondWith(final RequestResponseContext<?> context, final ConsumerB
 
     channelHandlerContext
       .writeAndFlush(replyBuffer)
-      .addListener(new ChannelFutureListener() {
-        @Override
-        public void operationComplete(final ChannelFuture future) throws Exception {
-          if (future.isSuccess()) {
-            logger.debug("Reply sent");
-          } else {
-            logger.error("Failed to send reply", future.cause());
-          }
-          if (closeFollowing) {
-            channelHandlerContext
-              .close()
-              .addListener(closeFuture -> {
-                logger.debug(">>>>> NettyInboundHandler::respondWith(): " + instanceId + " NAME: " + contextInstanceId(channelHandlerContext) + " : CLOSED");
-            });
-          }
+      .addListener((ChannelFutureListener) future -> {
+        if (future.isSuccess()) {
+          logger.debug("Reply sent");
+        } else {
+          logger.error("Failed to send reply", future.cause());
+        }
+        if (closeFollowing) {
+          future
+            .channel()
+            .close()
+            .addListener(closeFuture -> {
+              logger.debug(">>>>> NettyInboundHandler::respondWith(): " + instanceId + " NAME: " + contextInstanceId(channelHandlerContext) + " : CLOSED");
```

Comment:
@alexguzun I think if would be good to test `isSuccess()` and log differently if the close fails. Once the Heroku side is confirmed working we can eliminate a number of logs.

Reply:
Changed